public/js/selectors/index.js

```diff
@@ -3,10 +3,33 @@ import memoize from "lodash/memoize";
 
 // If the state object is the same then just return the same obj
 export const getFilters = memoize(({ filters }) => ({
-  ...filters,
-  values: {
-    ...filters.values,
-    startDate: moment(filters.values.startDate),
-    endDate: moment(filters.values.endDate),
-  }
+    ...filters,
+    values: {
+        ...filters.values,
+        startDate: moment(filters.values.startDate),
+        endDate: moment(filters.values.endDate)
+    }
 }));
+
+const getArticleWordCounts = ({ articleWordCounts }) => articleWordCounts;
```

Comment:
Is the purpose of this? Is it just to extract `articleWordCounts` from an object?

Reply:
Yeah, it's just means if we ever change where that's stored in the state we only have one place to change it ... it will also get used more for memoizing in my next branch 🙊